docs/graphql/manual/guides/postgres/functions.rst

```diff
@@ -0,0 +1,79 @@
+.. meta::
+   :description: Use Postgres functions with Hasura
+   :keywords: hasura, docs, postgres, functions
+
+.. _postgres_functions:
+
+Postgres functions
+==================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+
+`Postgres functions <https://www.postgresql.org/docs/current/sql-createfunction.html>`__ allow you to customize your database schema by defining a set of operations that can include several statements such as declarations, assignments and conditional workflows. 
+Postgres functions are similar to views but are used when we have arguments.
+
+.. note::
+
+  For more information on Postgres functions, please refer to the `Postgres documentation <https://www.postgresql.org/docs/current/sql-createfunction.html>`__.
+
+Examples
```

Comment:
It seems both examples are trigger functions, we should perhaps have an example of a trackable function with some arguments? (can copy straight from schema section in docs)

Reply:
In fact, since we also have a section on `triggers`, we should probably not talk about trigger functions at all.